lib/tftp/src/tftp.erl

```diff
@@ -350,7 +350,7 @@ matching regexp is found.
 -spec read_file(RemoteFilename, LocalFilename, Options) ->
         {ok, LastCallbackState} | {error, Reason} when
     RemoteFilename    :: file:filename(),
-    LocalFilename     :: file:filename_all(),
+    LocalFilename     :: file:filename_all() | binary,
```

Comment:
I don't understand this, filename_all already includes binary?

-type filename_all() :: [string](https://www.erlang.org/doc/apps/erts/erlang#t:string/0)() | (RawFilename :: [binary](https://www.erlang.org/doc/apps/erts/erlang#t:binary/0)()).

Reply:
Yes, but it's type binary(), tftp:read_file also handles specifically atom 'binary'.

Reply:
https://www.erlang.org/doc/apps/tftp/tftp.html#read_file/3

> If LocalFilename is the atom binary, tftp_binary is used as callback module. It concatenates all transferred blocks and returns them as one single binary in LastCallbackState.

Reply:
Aha, I missed that is was the atom can you qoute it so that it is more obvoius that is an atom.
i..e `'binary'`

Reply:
We normally quote atoms in specs so that they are more obvious, in other applications at least.

Reply:
Ok I will change that.